c7n/reports/csvout.py

```diff
@@ -263,44 +266,56 @@ def record_set(session_factory, bucket, key_prefix, start_date):
     """
 
     s3 = local_session(session_factory).client('s3')
-    marker = key_prefix.strip("/") + "/" + start_date.strftime(
-        '%Y-%m-%d-00') + "/resources.json.gz"
+
+    today = date.today()
+    current_date = start_date.date()
+    date_folders = []
```

Comment:
Right, sorry that's what I meant. Didn't realize it would come out of the 03 folder and back up to the 04 folder. I'll test that out and update if it works.